mscore/pluginCreator.cpp

```diff
@@ -480,7 +497,7 @@ void PluginCreator::newPlugin()
       created = true;
       QString s(
          "import QtQuick 2.0\n"
-         "import MuseScore 1.0\n"
+         "import MuseScore 3.0\n"
```

Comment:
This part might better be in a separate commit, as this applies to master only, the other stuff might apply to 2.2 too.
But I'd rather have @lasconic voice his opinion here ;-)

Reply:
Thanks for the heads up, I can fix that when cherry picking, no pb. So no need for 2 commits